README.md

```diff
@@ -24,10 +24,65 @@ A sample implementation of this can be found in the file `sample-main.go`
 and `pkg/sample-cmd` directory.  You'll want to have the equivalent files
 in your project.
 
-### A note on Dependencies
+### Example for developing your own Custom Metric Server API
+This is an example of how to vendor this project and setup your own provider.  This example uses [glide](https://github.com/Masterminds/glide) but you can use any other [dependency management tool](https://github.com/golang/go/wiki/PackageManagementTools) of your choice.
 
-You'll need to `glide install` dependencies before you can use this
-project.
+#### Create your new project:
```

Comment:
I like this, but perhaps we should move it to a separate file, like `docs/getting-started.md`, and the link here in a clear position so people see it.

Reply:
sounds good!